.github/workflows/commit.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
     strategy:
       fail-fast: false  # don't fail fast as sometimes failures are arch/OS specific
       matrix:  # Use versions consistent with wazero's Go support policy.
-        os: [ubuntu-22.04, macos-12, windows-2022]
+        os: [ubuntu-22.04, macos-11, macos-12, windows-2022]
```

Comment:
the macos-11 run was successful but I am going to remove it from CI, I just wanted to give it a go for double-check

Reply:
thanks!